### PR TITLE
Use PyGithub library instead of direct using of github api through requests

### PR DESCRIPTION
**Pull request by xobb1t**
What you think about it, @yumike ?

### CODE REVIEW

File: project/lint/ghclone.py [hunk 1]
```diff
@@ -31,34 +34,31 @@ def tempdir(root=None):
 @contextmanager
 def clone(url):
     with tempdir(root=CONFIG['CLONES_ROOT']) as path:
-        tarball_path = _download_tarball(url, path)
+        repo_owner, repo_name = url.split('/')
+        tarball_path = _download_tarball(repo_owner, repo_name, path)
```
Comment (yumike) [inline project/lint/ghclone.py]: Maybe, we should do it in `_download_tarball`? These vars are needed only for `repo` object creation.

Comment (xobb1t) [inline project/lint/ghclone.py]: hm, yep.

Comment (xobb1t) [inline project/lint/ghclone.py]: I think, we will change `clone` arguments someday, but now I will split `url` in `_download_tarball`.

**Comment by yumike** [comment]
Love it, let's merge!

**Comment by xobb1t** [comment]
:flushed:

**Pull request merged**